pkg/plugins/client.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 const (
-	defaultTimeOut = 30
+	defaultTimeOut     = 30
+	defaultHTTPTimeOut = 3
```

Comment:
It's because the http client is successfully connected to the server. It's the server that didn't response to the http request.